services/comment_service.py

```
@@ -0,0 +1,100 @@
+
+from fastapi import Depends, HTTPException,status
+
+from common.security import encode_id
+from repositories.comment_repository import CommentRepository
+from repositories.post_repository import PostRepository
+from repositories.user_repository import UserRepository
+from schemas.comment import CommentCreateRequest, CommentUpdateRequest
+from datetime import datetime, timezone
+
+class CommentService:
+    def __init__(self,comment_repository: Annotated[CommentRepository,Depends(CommentRepository)],
```

Ruff (0.14.13)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)